src/components/issues/TaskManagementModal.tsx

```diff
@@ -5,6 +5,7 @@ import Modal from '../Modal';
 import TaskRequestForm from './TaskRequestForm';
 import ActionForm from './ActionForm';
 import { TaskData, TaskRequestData } from '@/components/issues/constants';
+import Image from 'next/image';
```

Comment:
why are we using images here, we could find these easily in react-icons

Reply:
These images are from [RDS design system](https://www.figma.com/file/9ZIFD1BFFoO6bHVSSunxGr/Icon-System?type=design&node-id=0-1&mode=design&t=WdiWYzRg3b52rm6R-0). Not sure if we should replace these with similar react icons.

Reply:
yes there are similar icon available

Reply:
it's okay can be updated later